src/explore-education-statistics-admin/src/components/AdminDashboardPublicationsTab.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { Dictionary } from '@common/types';
+import Accordion from '@common/components/Accordion';
+import AccordionSection from '@common/components/AccordionSection';
+import { Publication } from '@admin/services/publicationService';
+import groupBy from 'lodash/groupBy';
+import AdminDashboardPublications from './AdminDashboardPublications';
+import Link from './Link';
+
+interface AdminDashboardPublicationsTabProps {
+  publications: Publication[];
+  noResultsMessage: string;
+}
+
+const AdminDashboardPublicationsTab = ({
+  publications,
+  noResultsMessage,
+}: AdminDashboardPublicationsTabProps) => {
+  const createThemeTopicTitleLabel = (publication: Publication) =>
+    `${publication.topic.theme.title}, ${publication.topic.title}`;
+
+  const publicationsByThemeAndTopic: Dictionary<Publication[]> = groupBy(
+    publications,
+    createThemeTopicTitleLabel,
+  );
+
+  const themesAndTopics = Object.keys(publicationsByThemeAndTopic);
+
+  if (themesAndTopics.length === 0) {
+    return <div className="govuk-inset-text">{noResultsMessage}</div>;
+  }
+
+  const publicationsForThemeTopic = (themeTopic: string) =>
+    publicationsByThemeAndTopic[themeTopic];
+
+  const themesAndTopicsSections = themesAndTopics.map(themeTopic => (
```

Comment:
Bit of a nitpick here, but would just inline this into the return expression as you're already doing most of the work in this variable anyway and you're not re-using in multiple places.

Generally, I would say prefer to inline JSX, especially when the component isn't particularly complex. In more complicated cases, then consider just splitting out into smaller components instead of doing it as separate fragment variables.

Reply:
Fine by me.  Done